keepassxc-browser/options/options.html

```diff
@@ -5,23 +5,24 @@
       <meta charset="UTF-8">
       <link rel="stylesheet" href="bootstrap.min.css" />
       <link rel="stylesheet" href="options.css" />
-      <script type="text/javascript" src="../browser-polyfill.min.js"></script>
-      <script type="text/javascript" src="../global.js"></script>
-      <script type="text/javascript" src="../jquery-3.3.1.min.js"></script>
-      <script type="text/javascript" src="bootstrap.min.js"></script>
-      <script type="text/javascript" src="options.js"></script>
+      <script src="../browser-polyfill.min.js"></script>
+      <script src="../global.js"></script>
+      <script src="../jquery-3.3.1.min.js"></script>
```

Comment:
Hu? Was not this PR about removing jQuery?

```suggestion
```

Reply:
I decided to leave jQuery for options and popup pages until there's a version of Bootstrap released that uses vanilla JavaScript. (Seems it could be released with version 5).